### DISCUSSION TITLE
OneDrive - Request failed, status 400

### DISCUSSION THREAD

**Discussion by Spotty10** [remotely-save/remotely-save#900] [Q&A]
Hi
Remotely Save has been working fantastically across three devices (PC, iphone, ipad). I had unrelated issues with my vault which resulted in me starting from scratch.
In the process I reinstalled Remotely Save on the three devices.
Now I cannot Auth the app with Microsoft…
The “Connecting to OneDrive… Please DO NOT close this modal.” sits open indefinately, and my iPad showed an error saying “Error: Request failed, status 400”

I have without success: 
- removed via consent and reconnected in my MS account
- deleted and reinstalled Obsidian off all devices
- tried a new vault

I’m unsure if this has triggered a lockout for the app with MS

Any ideas would be greatly appreciated

**Comment by bdabyjg** [comment]
I have this same problem too.

**Reply by brauer5** [reply]
Same issue here on all devices, any fix??

**Comment by Mylexstar** [comment]
please sign your onedrive to verify

**Reply by bdabyjg** [reply]
Hi, I have authorized and logged into my OneDrive account multiple times. Each time, it shows a successful login, but when it redirects back to Obsidian, it keeps displaying “Connecting to OneDrive… Please DO NOT close this modal.” The problem is that I have ONE empty folder. Additionally, I have tried uninstalling and reinstalling the plugin, but it still doesn’t work.

**Reply by XNDR1932** [reply]
Hello,

it's a little late but i came across this because i am having the same problem. Have you found a solution by any chance?

**Reply by bdabyjg** [reply]
> Hello,
> 
> it's a little late but i came across this because i am having the same problem. Have you found a solution by any chance?

Hi, here is how I fixed it: You might need to wait for a few weeks or a month, and then re-authorize it. I did this, and it worked for me.

**Reply by LuHavery** [reply]
你好，我找到解决办法了，问题的根源在于免费版本的插件不能读取onedrive的根目录，只能读取OneDrive里位于remotely-save文件夹里的文件，我建议你在obsidian把新文件创建在remotely-save的文件夹里，再点击鉴权。我是这样解决的，搞了一晚上终于成功了，希望对你有帮助 。不用客气

**Comment by Yzm079** [comment]
ok，我根据前面老哥的方法，成功连接好了。我预先将我的remotely-save文件放在了app文件下，导致每次链接他都给我生成一个新的remotely-save1，我将remotely-save移走后app文件清空，重新连接后生成remotely-save文件，此时我将原来remotely-save文件（即我预先保留的那个文件）下的笔记移动到生成的remotely-save中，再点击链接，成功了
